build/Jamfile.v2

```diff
@@ -1,7 +1,24 @@
-# Copyright (c) 2002-2020 CrystalClear Software, Inc.
+# Copyright (c) 2002-2005 CrystalClear Software, Inc.
 # Use, modification and distribution is subject to the 
 # Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or http://www.boost.org/LICENSE_1_0.txt)
 #
-# date_time is now header only -- this file is obsolete
+# date_time is now header only -- this file provides a stub for backward compatibility
 #
+
+
+project boost/date_time
+    : requirements
+        <define>DATE_TIME_INLINE
```

Comment:
Is this define necessary? (And similarly in usage-requirements.)

Reply:
No it's not -- it's legacy from what was there previously.  I'll clean it up when I clean up a bunch of other now dead code.  Thx!